### PR TITLE
Editorial and minor substantive changes

### PR DESCRIPTION
**Pull request by wseltzer**
s/need/want/
update WG names to match current groups
remove inaccurate PING questionnaire link
there's no reason for a web page to link via bit.ly
update Patent Policy link
https everywhere

### CODE REVIEW

**Review by plehegar — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -137,9 +137,10 @@ <h1 id="title">*PROPOSED* Web Performance Working Group Charter</h1>
 
       <section id="scope" class="scope">
         <h2>Scope</h2>
-        <p>Web developers are building sophisticated applications where application performance is a critical feature. Web developers need the ability to observe the performance characteristics of their applications, and they want the ability to write more efficient applications, using well-defined interoperable methods.</p>
+        <p>Web developers are building sophisticated applications where application performance is a critical feature. Web developers want the ability to observe the performance characteristics of their applications, and they want the ability to write more efficient applications, using well-defined interoperable methods.
+        Their methods must be both secure and privacy-enabling by design, based on the current Web browser security model.</p>
```
Comment (igrigorik) [inline index.html]: In principle, I agree with the last sentence, but there's some nuance here I'm not convinced about.. 

- What specifically constitutes a "secure" performance API? Scoped to secure origins only, something else?
- Ditto for "privacy preserving". Some API's _may_ expose new bits and that tradeoff is made on a feature by feature basis, based on detailed analysis of the tradeoffs. 

If the intent is to highlight that expose APIs / information must respect web browser security model (which I agree with), I'd propose we reword this as.. 

> "... using well-defined interoperable methods that conform to the current Web browser security model."

WDYT?

File: index.html [hunk 2]
```diff
@@ -237,7 +238,7 @@ <h3>
 
               <p>Note: after Level 2 of Navigation Timing is ratified, future development of new Navigation Timing features will be done within Resource Timing; Navigation Timing will be merged with Resource Timing.</p>
               <p>Latest publication: <a href="https://www.w3.org/TR/2017/WD-navigation-timing-2-20171205/">05 December 2017</a><br>Reference Draft:
-<a href="http://www.w3.org/TR/2015/WD-navigation-timing-2-20150120/">http://www.w3.org/TR/2015/WD-navigation-timing-2-20150120/</a><br>
+<a href="https://www.w3.org/TR/2015/WD-navigation-timing-2-20150120/">https://www.w3.org/TR/2015/WD-navigation-timing-2-20150120/</a><br>
```
Comment (igrigorik) [inline index.html:241]: @plehegar @siusin how did we derive these refs?

We have a newer publication @ https://www.w3.org/TR/2017/WD-navigation-timing-2-20171205/

Ditto for most of the other specs below.

Comment (siusin) [reply index.html:241]: How about using the refs to the latest published version? i.e., https://www.w3.org/TR/navigation-timing-2/

File: index.html (hunk 1 above)
Comment (wseltzer) [reply index.html]: @igrigorik I like your modification. The point is to highlight the tradeoffs for careful evaluation.

File: index.html (hunk 2 above)
Comment (wseltzer) [reply index.html:241]: The "Reference Draft" is defined by Process, https://www.w3.org/2018/Process-20180201/#WGCharter for Patent Policy purposes. It's distinct from the "Latest Publication" draft from which the group will continue its work.

File: index.html (hunk 1 above)
Comment (igrigorik) [reply index.html]: 👍 

I assume you'll make the text update? ty ahead of time!

Comment (wseltzer) [reply index.html]: Yes, done.

**Review by igrigorik — APPROVED** [review]
lgtm, modulo outstanding question about updating spec link references — should we tackle that in a separate PR?

thanks Wendy!

**Comment by plehegar** [comment]
Those links are necessary for the patent policy requirements. Those are the links to the reference drafts so we can't really change them.
